### PR TITLE
Adds a notification for ghosts for enabling or disabling CTF

### LINKED ISSUE #15603: Instagib mode on ctf has no way to turn off
And the instagib shot completly ignores hardsuit shields(not sure if intended)

### PR DESCRIPTION
**Pull request by Shadowlight213**
Jump links were added.
You can click on the alert to jump to the controller, or click a link in chat to directly join the team.

Also added a way to switch back from instagib mode.
Fixes #15603

### CODE REVIEW

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -535,6 +535,7 @@
 				ctf_enabled = !CTF.ctf_enabled
 				CTF.ctf_enabled = !CTF.ctf_enabled
 			message_admins("[key_name_admin(usr)] has [ctf_enabled? "enabled" : "disabled"] CTF!")
+			notify_ghosts("CTF has been [ctf_enabled? "enabled" : "disabled"]!")
```
Comment (KorPhaeron) [inline code/modules/admin/secrets.dm]: Should probably provide them a link to jump to the CTF arena, or instructions on how to spawn

**Comment by octareenroon91** [comment]
Maybe make the beacons say, 'CTF active'

File: code/modules/admin/secrets.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/modules/admin/secrets.dm]: Or make every machine you're activating throw a ghost alert that is clickable

**Comment by lzimann** [comment]
fixes #15603

File: code/modules/awaymissions/capture_the_flag.dm [hunk 2]
```diff
@@ -211,9 +223,15 @@
 /obj/machinery/capture_the_flag/proc/instagib_mode()
 	for(var/obj/machinery/capture_the_flag/CTF in machines)
 		if(CTF.ctf_enabled == TRUE)
-			CTF.ctf_gear = CTF.instagib_gear
+			CTF.current_gear = CTF.instagib_gear
```
Comment (KorPhaeron) [inline code/modules/awaymissions/capture_the_flag.dm]: Couldn't you do CTF.gear = initial(CTF.gear) instead of adding current_gear and normal_gear?

Comment (Shadowlight213) [inline code/modules/awaymissions/capture_the_flag.dm]: That's actually a better way, yeah.

**Pull request merged**